data_sample/tests/test_login_example.py

```diff
@@ -0,0 +1,17 @@
+# -*- encoding: utf-8 -*-
```

Comment:
`-*- encoding: utf-8 -*-` is a format for emacs python default one is `coding=utf-8` better to use it.